ac_termination_protection/variables.tf

```diff
@@ -0,0 +1,65 @@
+variable "name" {
+  type        = string
+  description = "The name of the resource"
```

Comment:
Of which resource? The one being protected? Or the webhook deployment?

I know what it refers to as I've read the code, but let's be more clear here. Otherwise, the description is useless and one has to always read the code.